src/sidebar/helpers/version-data.js

```diff
@@ -1,37 +1,22 @@
 /**
  * @typedef {import('../../types/annotator').SegmentInfo} SegmentInfo
+ * @typedef {import('../store/modules/frames').Frame} Frame
  */
 
 /**
- * @typedef AuthState
+ * @typedef UserDetails
  * @prop {string|null} [userid]
  * @prop {string} [displayName]
  */
 
-/**
- * An object representing document metadata.
- *
- * @typedef DocMetadata
- * @prop {string=} documentFingerprint - Optional PDF fingerprint for current document
- */
-
-/**
- * An object representing document info.
- *
- * @typedef DocumentInfo
```

Comment:
Or, alternately, we could define a type here, but it should be based on (e.g. `pick`) props out of `Frame`, probably.